jvm/ml4ir-inference/pom.xml

```diff
@@ -0,0 +1,183 @@
+      </plugin>
+		<!-- disable surefire - - >
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.7</version>
+          <configuration>
+            <skipTests>true</skipTests>
+          </configuration>
+        </plugin>
+        <! - - enable scalatest
+        <plugin>
+          <groupId>org.scalatest</groupId>
+          <artifactId>scalatest-maven-plugin</artifactId>
+          <version>1.0</version>
+          <configuration>
+            <reportsDirectory>${project.build.directory}/surefire-reports</reportsDirectory>
+            <junitxml>.</junitxml>
+            <filereports>WDF TestSuite.txt</filereports>
+          </configuration>
+          <executions>
+            <execution>
+              <id>test</id>
+              <goals>
+                <goal>test</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin> -->
+              <!-- for fatjar -->
+      <plugin>
+	<groupId>org.apache.maven.plugins</groupId>
+	<artifactId>maven-assembly-plugin</artifactId>
+	<version>3.1.0</version>
+	<configuration>
+	  <descriptorRefs>
+	    <descriptorRef>jar-with-dependencies</descriptorRef>
+	  </descriptorRefs>
+	  <finalName>uber-ml4ir-parent-0.0.1-SNAPSHOT</finalName>
+	  <appendAssemblyId>false</appendAssemblyId>
+	</configuration>
+	<executions>
+	  <execution>
+	    <id>assemble-all</id>
+	    <phase>package</phase>
+	    <goals>
+	      <goal>single</goal>
+	    </goals>
+	  </execution>
+	</executions>
+      </plugin>
+      <plugin>
+	<groupId>org.apache.maven.plugins</groupId>
+	<artifactId>maven-jar-plugin</artifactId>
+	<configuration>
+	  <archive>
+	    <manifest>
+	      <addClasspath>true</addClasspath>
+	      <mainClass>fully.qualified.MainClass</mainClass>
+	    </manifest>
+	  </archive>
+	</configuration>
+      </plugin>
+    </plugins>
+    <pluginManagement>
+      <plugins>
+	<!--This plugin's configuration is used to store Eclipse m2e settings 
+	    only. It has no influence on the Maven build itself. -->
+	<plugin>
+	  <groupId>org.eclipse.m2e</groupId>
+	  <artifactId>lifecycle-mapping</artifactId>
+	  <version>1.0.0</version>
+	  <configuration>
+	    <lifecycleMappingMetadata>
+	      <pluginExecutions>
+		<pluginExecution>
+		  <pluginExecutionFilter>
+		    <groupId>net.alchim31.maven</groupId>
+		    <artifactId>scala-maven-plugin</artifactId>
+		    <versionRange>[3.3.1,)</versionRange>
+		    <goals>
+		      <goal>compile</goal>
+		      <goal>testCompile</goal>
+		    </goals>
+		  </pluginExecutionFilter>
+		  <action>
+		    <execute />
+		  </action>
+		</pluginExecution>
+	      </pluginExecutions>
+	    </lifecycleMappingMetadata>
+	  </configuration>
+	</plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
+  <dependencies>
+    <dependency>
+    	<groupId>org.tensorflow</groupId>
+    	<artifactId>tensorflow</artifactId>
```

Comment:
Can we add an explicit version number here?

Reply:
I think it's in the parent pom already (you have to specify)